### PR TITLE
switch to clang 3.8 & ignore include sort order

### PR DESCRIPTION
**Pull request by Stunkymonkey**
my system is an arch-linux, so i am having a newer clang-version
everytime i make an `clang-format` i have to revert these changes.

so it would be easier for me if the includes have the right order

### CODE REVIEW

**Comment by orestisf1993** [comment]
We could also just set `SortIncludes` like in i3:
https://github.com/i3/i3/blob/6339427f017e1265a022e6537e16a8b4a921e52f/.clang-format#L11

**Comment by Stunkymonkey** [comment]
would be a better fix definitely

**Comment by orestisf1993** [comment]
I think clang-format-3.9 is also available

File: i3status.c [hunk 1]
```diff
@@ -44,7 +44,8 @@
     CFG_STR_CB("align", NULL, CFGF_NONE, parse_align)
 
 #define CFG_COLOR_OPTS(good, degraded, bad)             \
-    CFG_STR("color_good", good, CFGF_NONE),             \
+    CFG_STR("color_good", good, CFGF_NONE)              \
+    ,                                                   \
```
[the remark this answers is not in the document]
Comment (Stunkymonkey) [reply i3status.c]: for some reasons travis will complain...
or do you prefer switching to clang-format-3.9?

[the remark this answers is not in the document]
Comment (Airblader) [reply i3status.c]: Let's not worry too much about it.

**Pull request merged**